src/views/SignupTypeSelector.vue

```diff
@@ -60,57 +62,29 @@ export default class SignupTypeSelector extends Vue {
 </script>
 
 <style scoped>
-    .page-body {
-        background: #fafafa;
-        overflow: auto;
-        padding-bottom: 2rem;
-    }
-
-    p {
-        margin: 4rem;
-        font-size: 2rem;
-        line-height: 1.3;
-        opacity: 0.7;
-    }
-
-    button {
-        font-weight: normal;
-        text-transform: none;
+    .keyguard-button,
+    .ledger-button {
+        display: block;
         text-align: left;
-        letter-spacing: normal;
         padding: 2rem;
         border-radius: 0.5rem;
 
-        margin: 0 4rem 2rem;
-        border: solid 2px #e5e5e5;
-        box-shadow: unset;
-        width: auto;
-        background: transparent;
+        margin: 2.75rem 0;
+        border: none;
         position: relative;
+        font-family: inherit;
 
-        max-width: unset;
-        height: auto;
-    }
-
-    button h2 {
-        font-size: 2.25rem;
-        line-height: 2.5rem;
-        font-weight: 600;
-        margin: 0 0 0.8rem;
-    }
-
-    button p {
-        margin: 0;
+        cursor: pointer;
     }
 
-    button.purple-background {
-        background: #724ceb;
-        color: white;
-        border: none;
+    .ledger-button {
+        border: solid 2px #e5e5e5;
```

Comment:
color

Reply:
This is a one-off color not specified by the theme. It's only in the designs for the accounts manager so far.

Reply:
Should we do that??

Reply:
Just quickly searched for `color: ` and found as an example `SignupSuccess.vue`. It has twice hard-code the same color in just this one file. I think we should only have a fixed set of constants representing the colors in the style guide. I can do a fork+PR replacing all the colors. Let me know what you think is best to do.

Reply:
Just replaced the last hardcoded color value here: https://github.com/nimiq/accounts/pull/55/commits/9bfc1f3e530cfda01e99221893bedb77855a92f1. The other places you found must have been in outdated code.